mmcls/datasets/imagenet.py

```diff
@@ -49,11 +49,8 @@ def get_samples(root, folder_to_idx, extensions):
     """
     samples = []
     root = os.path.expanduser(root)
-    for folder_name in sorted(os.listdir(root)):
+    for folder_name in sorted(list(folder_to_idx.keys())):
```

Comment:
After the change, we need to assume all folders in `folder_to_idx` exist.
Considering it's an inner function, and only be used after `find_folders`, I think it's a reasonable assumption.